t/file-writelines.neon

```diff
@@ -0,0 +1,31 @@
+IMPORT file
+
+VAR lines: Array<String> := ["Hello", "Neon", "", "World!"]
+
+TRY
+    file.writelines("linetest.txt", lines)
+EXCEPTION file.FileOpenError
```

Comment:
Same comments as above here for all the exception catches. You aren't expecting any of them.